Formula/apk-patch-size-estimator.rb

```diff
@@ -0,0 +1,19 @@
+class ApkPatchSizeEstimator < Formula
+  desc "Estimates the size of Play patches / gzipped APKs."
+  homepage "https://github.com/googlesamples/apk-patch-size-estimator"
+  url "https://github.com/googlesamples/apk-patch-size-estimator/archive/0.2.tar.gz"
+  sha256 "cbab54effbbbb2abf3ddb58cf004e8ee0e38b520fbc5266f08a5fd6a8cf9c9ff"
+  depends_on "bsdiff"
+
+  def install
+    libexec.install Dir["*"]
+
+    Dir.glob("#{libexec}/*.py") do |script|
```

Comment:
So I can't seem to figure out how to do it - but the `lib` folder inside `libexec` needs to be copied into `bin` (and remain as a subdirectory). Is there a way to symlink or something that directory from one to the other?